src/pudl/metadata/resources/allocate_gen_fuel.py

```diff
@@ -10,6 +10,24 @@
     "type": "custom",
     "description": "This downscaling process used to create this table does not distinguish between primary and secondary energy_sources for generators (see below for implications).",
 }
+UPSTREAM_ALLOCATION_CONTEXT = """The net generation and fuel consumption allocation
+method PUDL employs begins with the following context of the originally reported
+EIA-860 and EIA-923 data:
+* The :ref:`core_eia923__monthly_generation_fuel` table is the authoritative source of
+  information about how much generation and fuel consumption is attributable to an
+  entire plant. This table has the most complete data coverage, but it is not the most
+  granular data reported.
+* The :ref:`core_eia923__monthly_generation` table contains the most granular net
+  generation data. It is reported at the ``plant_id_eia``, ``generator_id`` and
+  ``report_date`` level. This table includes only ~40% of the total MWhs reported in
+  the :ref:`core_eia923__monthly_generation_fuel` table.
+* The :ref:`core_eia923__monthly_boiler_fuel` table contains the most granular fuel
+  consumption data.  It is reported at the boiler/prime mover/energy source level. This
+  table includes only ~40% of the total MMBTUs reported in the
+  :ref:`core_eia923__monthly_generation_fuel` table.
+* The :ref:`core_eia860__scd_generators` table provides an exhaustive list of all
+  generators whose generation is being reported in the
+  :ref:`core_eia923__monthly_generation_fuel` table."""
```

Comment:
i ripped most of this from the module docs from `pudl.analysis.allocate_gen_fuel.py`. imho the [module docs](https://catalystcoop-pudl.readthedocs.io/en/latest/autoapi/pudl/analysis/allocate_gen_fuel/index.html) over there are pretty descriptive of what the heck is going on. it is effectively trying to be a methodology page. this feels a lil oos but might be a good idea to convert this into a more user-focused page.